CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+### 0.6.0 (2020-01-14)
+
+* PFS: Fix DB schema to not override channels (https://github.com/raiden-network/raiden-services/issues/693)
```

Comment:
must be scheme, right?

Reply:
schema is the right word: https://en.wikipedia.org/wiki/Database_schema